flexmeasures/data/tests/test_scheduling_repeated_jobs.py

```diff
@@ -197,9 +198,11 @@ def test_scheduling_multiple_triggers(
     soc_start = 2.5
 
     assert (
-        DataSource.query.filter_by(name="FlexMeasures", type="scheduling script")
-        .where()
-        .one_or_none()
+        db.session.execute(
+            select(DataSource)
+            .filter_by(name="FlexMeasures", type="scheduling script")
+            .where()
```

Comment:
Does this pass without the `.where()`? I have a hunch it is not doing anything.

Reply:
yes, I will drop it.